openapi.yaml

```diff
@@ -16015,13 +16034,47 @@ components:
         type:
           type: string
           description: A Linode Type for all of the nodes in the Node Pool.
-          example: "g6-standard-4"
+          example: g6-standard-4
         count:
           type: integer
           description: The number of nodes in the Node Pool.
           minimum: 1
           maximum: 100
           example: 6
+        disks:
+          type: array
+          description: >
+            This Node Pool's custom disk layout. Each item in this array will create a new disk
+            partition for each node in this Node Pool.
+
+              * The custom disk layout is applied to each node in this Node Pool.
```

Comment:
What is worth noting here is:

* If this field is omitted, the Node will have a default disk layout consisting of a single partition. A custom layout is only required for specific use cases.

Reply:
Added! Thanks for the suggestion!